ERROR_MESSAGES.md

```diff
@@ -8,14 +8,14 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 3 : CT - Unexpected exception occurred during conversion. Please contact the Service Center for assistance via phone at 1-866-288-8292 or TTY: 1-877-715-6222, or by emailing QPP@cms.hhs.gov
 * 4 : CT - Unexpected exception occurred during encoding. Please contact the Service Center for assistance via phone at 1-866-288-8292 or TTY: 1-877-715-6222, or by emailing QPP@cms.hhs.gov
 * 5 : CT - The file is not a QRDA-III XML document. Please ensure that the submission complies with the `(Submission year's)` implementation guide. `(Implementation guide link)`
-* 6 : CT - The measure reference results must have a single occurrence of the recognized measure GUID `(Provided measure id)` is invalid. Did you intend to send one of these `(Valid measure id suggestions)`?
+* 6 : CT - The measure GUID supplied `(Provided measure id)` is invalid. Please see the 2017 IG https://ecqi.healthit.gov/system/files/2018_CMS_QRDA_III_Eligible_Clinicians_and_EP_IG_11272017_508_0.pdf#page=88 for valid measure GUIDs.
```

Comment:
This may or may not be an issue but the link doesn't correctly link me to the eCQM Uuid list using Safari. Chrome works fine.